db/migrate/20170417192258_create_interaction_claims_publications_table.rb

```diff
@@ -0,0 +1,15 @@
+class CreateInteractionClaimsPublicationsTable < ActiveRecord::Migration
+	def up
```

Comment:
The indentation in this file is off (too far indented). The standard is two spaces per indentation level.